state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1013,11 +1050,15 @@ pub mod pallet {
 				};
 
 			let gas_swap_id = if let Some(other_gas_asset) = other_gas_asset {
+				let execute_at =
```

Comment:
we should probably ensure both ccm swaps are in the same block by using a variable used by both here